python/tests/test_filesystem.py

```
@@ -186,7 +227,29 @@ def test_mkdir_flags():
   assert status.ok
   c.rm('/tmp/dir1/dir2')
   c.rm('/tmp/dir1')
-  
+
+def test_mkdir_p():
+  c = client.FileSystem(SERVER_URL)
+  status, response = c.mkdir_p('/tmp/dir1/dir2', flags=MkDirFlags.NONE)
```

Ruff (0.16.3)

[warning] Unpacked variable `response` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)